src/main/java/net/imagej/ops/image/ImageNamespace.java

```diff
@@ -58,7 +62,7 @@
 	@OpMethod(op = net.imagej.ops.image.ascii.DefaultASCII.class)
 	public <T extends RealType<T>> String ascii(final IterableInterval<T> image) {
 		final String result = (String) ops().run(
-				net.imagej.ops.Ops.Image.ASCII.class, image);
```

Comment:
several format changes in this file make it hard to review the actual changes. can you undo the format changes and commit only thing related to the functionality you added/changed?